app/views/volunteers/index.html.erb

```diff
@@ -24,12 +24,12 @@
             <input
               id="unassigned-vol-filter"
               type="checkbox"
-              data-value="No supervisor assignment"
```

Comment:
Why did this go away?

Reply:
@compwron Filtering is now done server-side, and the empty value is easier to handle.